### ISSUE TITLE
Storybook does not build with vite 2.7.0-beta.4

### ISSUE THREAD

**Issue by IanVS** [eirslett/storybook-builder-vite#171]
I attempted to update to the latest vite alpha, and found that storybook would not build correctly.  I got this error from rollup:

```
Could not resolve entry module (iframe.html).
ERR! Error: Could not resolve entry module (iframe.html).
```

It seems that we'll need to make an update to our logic in `resolveId`, because the full url is provided now, like `/Users/ianvs/code/project-root/iframe.html`. rather than just `iframe.html` like we are checking for (https://github.com/eirslett/storybook-builder-vite/blob/main/packages/storybook-builder-vite/code-generator-plugin.js#L47)

**Comment by IanVS** [comment]
I've tracked this down to https://github.com/vitejs/vite/pull/5601, looks like `rollupOptions.input` paths are now being resolved.  So maybe we were just relying on a bug, previously.  This should be a fairly straightforward fix, and I think we can do it in a backwards-compatible way.

**Comment by IanVS** [comment]
https://github.com/eirslett/storybook-builder-vite/releases/tag/v0.1.11 has been released on npm, which supports vite 2.7.0.
